src/core/unittest/TicketTest.cpp

```diff
@@ -103,6 +103,28 @@ struct TransportParamsScope {
     }
 };
 
+struct TicketScope {
+    uint8_t* p = nullptr;
+
+    ~TicketScope() {
+        reset();
+    }
+
+    uint8_t** operator&() = delete;
```

Comment:
This operator isn't defined anyway, so there is no reason to delete it. the default operator& will return a TicketScope*.

Reply:
Why not explicitly discourage the usage of `(uint8_t**) operator&()`? Won't that be covered by this?